vpc/service/db/migrations/bindata.go

```diff
@@ -565,6 +565,24 @@ func _37_add_ipv4_cidr_up_sql() ([]byte, error) {
 	)
 }
 
+var __38_static_v2_down_sql = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\x0a\x0e\x71\x0c\x0a\x51\x08\x09\x72\xf4\x0b\x76\x74\x0e\xf1\xf4\xf7\x53\xb0\xe6\x72\x09\xf2\x0f\x50\xf0\xf4\x73\x71\x8d\x50\xc8\x2c\x88\x4f\x4c\x49\x29\x4a\x2d\x2e\x4e\x2d\x8e\xcf\x2c\x28\x33\x83\xf2\xe2\x8b\x4b\x93\xf2\x52\x4b\xe2\x33\x53\xe2\x4b\x33\xf3\x52\x52\x2b\xac\xb9\x1c\x7d\x42\x5c\x83\x14\x42\x1c\x9d\x7c\x5c\x51\xb4\x29\xa4\x14\xe5\x17\x28\x24\xe7\xe7\x94\xe6\xe6\x29\x20\x19\x41\xa4\x8e\x32\x93\x82\xa2\xd4\xb4\x4c\x62\x2d\x28\x33\x83\x29\x77\xf6\xf7\xf5\xf5\x0c\xb1\x06\x04\x00\x00\xff\xff\x71\x4b\x14\xc2\xe1\x00\x00\x00")
+
+func _38_static_v2_down_sql() ([]byte, error) {
+	return bindata_read(
+		__38_static_v2_down_sql,
+		"38_static_v2.down.sql",
+	)
+}
+
+var __38_static_v2_up_sql = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\xb4\x8f\x31\x6f\x83\x30\x10\x46\x77\x7e\xc5\x8d\x89\xd4\x31\x62\xc9\x84\x32\x65\x48\x22\x25\xec\x96\x83\x3f\xaa\x53\xd0\x41\xcf\x67\x44\xff\x7d\x15\x44\x23\x58\x5a\x3a\xd4\x9b\x2d\xbf\xa7\xef\xdd\xca\xe2\x5a\x52\x79\x2d\xce\xb7\xe2\x50\x1e\x2f\x67\xda\x67\x99\x6f\x0c\x4a\xe6\xef\x0d\x88\x3b\xe7\x43\x50\xc4\x88\x98\x11\x11\xf9\x10\x88\xbb\x3e\x9f\x5e\x89\x05\xb6\x86\xe9\xf3\x4e\x51\xf3\x40\x86\x61\x1d\xb0\x5b\x02\x95\xc2\x1b\x28\x09\x7f\x24\x10\x4b\xc0\xb0\x00\xdd\x6c\x95\x8b\xe9\x2e\x30\xc7\xc1\xa5\xf1\xe7\x68\x6d\x65\x01\xd0\x66\x46\xbc\xd1\x0b\xd9\xae\x59\x57\xb5\x12\x4d\x3d\x8b\x2d\x47\x4c\x96\x8a\x83\x3a\x45\x84\xf6\xde\xb8\x95\xe8\xfa\xdd\xfc\xfe\x5c\x56\x3f\x46\xdb\xf3\xd4\xad\x82\xdf\x85\x1e\xf8\xa4\xcd\x77\xf8\x96\x14\x35\x14\x52\x21\xd2\x0f\xde\x97\x65\x6a\x0c\x68\x60\xa0\x08\x23\x49\x4d\xf3\x3f\x35\xf9\xea\x9a\xfc\x4f\x35\xf9\xef\x35\xd9\xe1\x72\x3a\x1d\xcb\xfd\x57\x00\x00\x00\xff\xff\xc2\x38\xbe\x9a\xbc\x02\x00\x00")
+
+func _38_static_v2_up_sql() ([]byte, error) {
+	return bindata_read(
+		__38_static_v2_up_sql,
+		"38_static_v2.up.sql",
```

Comment:
I don't see 38_static_v2.up.sql. Did you check it in?

Reply:
missed that. added